modules/auxiliary/scanner/ike/cisco_ike_benigncertain.rb

```diff
@@ -34,6 +34,9 @@ def initialize(info = {})
           [ 'URL', 'https://nvd.nist.gov/vuln/detail/CVE-2016-6415' ],
           [ 'URL', 'https://musalbas.com/2016/08/18/equation-group-benigncertain.html' ]
         ],
+      'Notes' => {
+        'AKA' => ['BENIGNCERTAIN']
```

Comment:
I didn't see a reference to `BENIGNCERTAIN` in either link on the description.

Reply:
It's literally in the module name :P